docs/04_glossary.md

```diff
@@ -33,15 +40,16 @@ The Go client doesn't use this.
 
 ### Decision
 Any action taken by the workflow durable function is called a decision. For example:
-scheduling an activity, canceling a child workflow, or starting a timer.
+scheduling an activity, canceling a child workflow, or starting a timer. A [decision task](completion) contains an optional list of decisions. Every decision is recorded in the [event history](#event-history) as an [event](#event)
```

Comment:
`[decision task](completion)` -> `[decision task](#decision-task)`?

Reply:
and please end the sentence with a period :)